modules/osdk-scorecard-config.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+[id="osdk-scorecard-config_{context}"]
+= Scorecard configuration
+
+The scorecard tool uses a configuration file that allows configuring internal
+plug-ins, as well as a few global configuration options.
+
+[id="osdk-scorecard-config-file_{context}"]
+== Configuration file
+
+The scorecard tool's configuration file default location is the
+`<project_dir>/.osdk-scorecard.*`. The following is an example of a
+YAML-formatted configuration file:
+
+.Scorecard configuration file
+[source,yaml]
+----
+scorecard:
+  output: json
+  plugins:
+    - basic: <1>
+        cr-manifest:
+          - "deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml"
+          - "deploy/crds/cache.example.com_v1alpha1_memcachedrs_cr.yaml"
+    - olm: <2>
+        cr-manifest:
+          - "deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml"
+          - "deploy/crds/cache.example.com_v1alpha1_memcachedrs_cr.yaml"
+        csv-path: "deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml"
+----
+<1> `basic` tests configured to test two CRs.
+<2> `olm` tests configured to test two CRs.
+
+From highest priority to least, the hierarchy of configuration methods for the
+global options that are also configurable using a flag is:
+
+Command arguments -> configuration file -> default
+
+The configuration file must be in YAML format. As the configuration file may be
+extended to allow configuration of all `operator-sdk` subcommands in the future,
+the scorecard's configuration must be under a `scorecard` subsection.
+
+[NOTE]
+====
+Configuration file support is provided by the `viper` package. For more info
+on how viper configuration works, see the `viper` package
+link:https://github.com/spf13/viper/blob/master/README.md[README].
+====
+
+[id="osdk-scorecard-config-args_{context}"]
+== Command arguments
+
+While most of the scorecard tool's configuration is done using a configuration
+configuration file, the following arguments can also be used:
```

Comment:
the following arguments can also be used -> you can also use the following arguments